### PR TITLE
Restructuring of code

### PR DESCRIPTION
**Pull request by sa501428**

### CODE REVIEW

File: python/straw/straw.py [hunk 1]
```diff
@@ -315,65 +350,64 @@ def readBlock(req, size):
...
+            temp = temp + 2
             for i in range(nPts):
-                row=int(i/w)
-                col=i-row*w
-                bin1=int(binXOffset+col)
-                bin2=int(binYOffset+row)
-                if (useShort==0):
-                    c = struct.unpack('<h',uncompressedBytes[temp:(temp+2)])[0]
-                    temp=temp+2
+                row = int(i / w)
```
Comment (zozo123) [inline python/straw/straw.py]: `@sa501428, your editor adds too many edits due to styling. First, this styling doesn't follow PEP's (https://www.python.org/dev/peps/pep-0008/). Second, it changes the history of `git blame` and will make a debugger life harder as well as the reviewers for this restructuring commit.

**Comment by sa501428** [comment]
done

File: python/straw/straw.py (hunk 1 above)
Comment (nchernia) [reply python/straw/straw.py]: This should be resolved now.

**Comment by nchernia** [comment]
Can you run the same checks as before re: commenting out if statement? I think there might have been a bug with row/col (also a bug in the C code) that we never caught since we had that additional check.

**Pull request merged**